src/server_manager/install_scripts/install_server.sh

```diff
@@ -378,14 +378,18 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
   PUBLIC_HOSTNAME=${FLAGS_HOSTNAME:-${SB_PUBLIC_IP:-$(curl -4s https://ipinfo.io/ip)}}
```

Comment:
Perhaps we can use the dig command here instead, or in addition to ipinfo.
I'm not sure whether dig is widely available though. There's also host and nslookup. We could pick whatever is more widely used, or just try all of them.

This could fix the Oracle problem

Reply:
I think this is a good idea for a followup PR.

Reply:
Sounds good.